src/components/ec-dropdown-search/ec-dropdown-search.story.js

```diff
@@ -1,84 +1,99 @@
-  text: 'text',
-};
-const user = {
-  clientName: 'Ebury Demo 2',
-  email: 'ebury.demo2@ebury.com',
-  gravatar: 'https://www.gravatar.com/avatar/e07fd6efc70ccc63bbc3a3e27b81b29e?d=mm&s=200',
-};
 const items = [
-  {
-    disabled: false,
-    disabledReason: 'This ac+44 (0) 207 197 2421.',
-    id: 'EBPCLI00004',
-    text: 'Ebury Demo',
-  },
-  {
-    disabled: false,
-    disabledReason: 'This ac+44 (0) 207 197 2421.',
-    id: 'EBPCLI00007',
-    text: 'Ebury Demo 2',
-  },
+  { text: 'Item 1' },
+  { text: 'Item 2' },
+  { text: 'Item 3', disabled: true, disabledReason: 'Is disabled for a reason' },
+  { text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod' },
 ];
 
-const stories = storiesOf('Ec dropdown Search', module);
-
 stories.add('basic', () => ({
-  components: { EcDropdownSearch },
+  components: { EcDropdownSearch, EcIcon },
+  data() {
+    return {
+      selectedItem: null,
+    };
+  },
   props: {
-    defaultItemSelected: {
-      default: object('defaultItemSelected', null),
-    },
-    huhaTaskName: {
-      default: text('url', 'applyDropdownAccountSelector'),
+    items: {
+      default: object('Items', items),
     },
-    huhaTrackOnGoogleAnalytics: {
-      default: boolean('huhaTrackOnGoogleAnalytics', true),
+    isSearchEnabled: {
+      default: boolean('isSearchEnabled', true),
     },
-    huhaTrackOnIntercom: {
-      default: boolean('huhaTrackOnIntercom', false),
+  },
+  methods: {
+    onItemSelected: action('Item selected'),
+  },
+  template: `
+    <div style="width: 100vw; height: 100vh;" class="ec-p--20">
+      <p v-if="selectedItem">Selected item: {{ selectedItem.text }}</p>
+      <p v-else>Selected item: None</p>
+      <ec-dropdown-search
+        :items="items"
+        :popover-options="{ placement: 'bottom-end' }"
+        :is-search-enabled="isSearchEnabled"
+        v-model="selectedItem"
+        @change="onItemSelected">
+        <a href="#" @click.prevent>
+          <span>Open</span>
+          <ec-icon name="simple-arrow-drop-down" :size="16" fill="currentColor" />
+        </a>
+      </ec-dropdown-search>
+    </div>
+  `,
+}));
+
+stories.add('in container with a dynamic width', () => ({
+  components: { EcDropdownSearch, EcIcon },
+  data() {
+    return {
+      selectedItem: null,
+    };
+  },
+  props: {
+    items: {
+      default: object('Items', items),
     },
-    itemFormat: {
-      default: object('itemFormat', itemFormat),
+    isSearchEnabled: {
+      default: boolean('isSearchEnabled', true),
     },
-    items: {
-      default: object('items', items),
+    paragraphText: {
+      default: boolean('paragraphText', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Lectus vestibulum mattis ullamcorper velit sed ullamcorper morbi tincidunt ornare. Consequat interdum varius sit amet mattis vulputate enim nulla. Eget mi proin sed libero enim sed faucibus turpis.'),
```

Comment:
I am assuming you haven't pushed yet. If not, don't forget to swap to text here :)